smelli/classes.py

```diff
@@ -313,28 +338,51 @@ def obstable(self, min_pull=0):
         due to correlations.
         """
         info = self._obstable_tree
-        info = self._obstable_filter_sort(info, min_pull=min_pull,
-                                          sortkey='pull', reverse=True)
+        subset = None
+        if sort_by == 'pull':
+            if ascending is None:
+                ascending = False
```

Comment:
What's that? Why is the default value not `False` then?

Reply:
Oh sorry, now I get it. It depends on what it is sorted by, OK.
Could you please add a few comments?

Reply:
Done in https://github.com/smelli/smelli/pull/1/commits/3f968936b1d471f26c63e697294e53110a6a4632.